R/courses.R

```diff
@@ -2127,17 +2135,8 @@ c_50 <- function(input_df=usheUtils::fake_course_df, with_intermediates=FALSE) {
 
   output_df <- input_df %>%
     # Calculate intermediate fields
-    mutate(c_delivery_model = case_when( instruction_method_code == "B" ~ 'B',
-                                       instruction_method_code == "V" ~ '?', # need to check
-                                       instruction_method_code == " " ~ "P", # need to check
-                                       instruction_method_code == "E" ~ 'H', # need to check
-                                       instruction_method_code == "R" ~ '?', # need to check
-                                       instruction_method_code == "I" ~ '?', # need to check
-                                       instruction_method_code == "P" ~ '?', # need to check
-                                       instruction_method_code == "H" ~ '?', # need to check
-                                       instruction_method_code == "O" ~ 'O',
-                                       instruction_method_code == "T" ~ 'T',
-                                       TRUE ~ instruction_method_code) ) %>%
+    mutate(c_delivery_model = if_else( instruction_method_code == "E", "E", NA_character_)) %>%
```

Comment:
@andreabr-dixie is "E" really the only valid instruction_method_code to send to USHE?